install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -1083,6 +1083,13 @@ data:
   envoy-log: {{ .Values.envoy.log.path | quote }}
 {{- end }}
 
+{{- if .Values.garp.enabled }}
+  # TODO(markpash): figure out the best way to expose this feature.
+  # Perhaps merge with Dylan's L2 helm values.
+  l2-announce-pods: "true"
+  garp-interface: {{ .Values.garp.interface | quote }}
+{{- end }}
+
```

Comment:
Could you point a PR or similar here for `Dylan's L2 helm values`?

Reply:
His PR is merged and so my config values now closely align with his. :)